doc/importing-template.md

```diff
@@ -0,0 +1,69 @@
+## How to import the template into Azure DevOps
+
+### Pre-requisites:
+
+[Create a DevOps Organizaton](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/create-organization?view=azure-devops#create-an-organization) or use an existing one. 
+
+### Process:
+
+1. Navigate to the [Azure DevOps Demo Generator](https://azuredevopsdemogenerator.azurewebsites.net/Account/Verify) and sign into your DevOps organization.
+2. Go to https://aka.ms/fta/iacmigrationtemplate, sign into your DevOps organization and you already will be in the correct template selection for IaC Migration (Please note that if you click on Choose template, this template is available under **FastTrack for Azure** tab as well)
+3. Select **Create Project** on the main page. 
+4. After the successfully provisioning of the template, go to the Project > Boards > Backlogs and on the top-right click on the engine (Configure team settings). Validate that Epics and Stories are selected and then click **Save and Close**. 
+
+![backlog-setting-0](/png/backlog-setting-0.png)
+
+
+![backlog-setting-1](/png/backlog-setting-1.png)
+
+5. Now change to view the Epics then you have all set, containing all Epics, User Stories and Tasks.
+
+![backlog-setting-2](/png/backlog-setting-2.png)
+
+### Customization:
+
+It is recommended to utilize Wave, Deliverable, and Activity terminology in naming the work items, so here is a guidance over the process of customization to have something different than Epics, Stories and Tasks:
+
+1. Click on the Azure DevOps logo in the top 
+2. Click on Organization Settings
+3. Go to Boards > Process
+4. Right click on Agile then click to Create inherited process
+5. Name as you want. In this case let's name as Migration, then click to Create process
```

Comment:
5. Choose the name of the process, in this example it is named Migration, and then click to `Create Process`